index.html

```diff
@@ -3,10 +3,68 @@
       <head>
         <meta charset="UTF-8">
         <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins">
         <link rel="stylesheet" href="styles/main.css">
-        <title>JT</title>
+        <title>JT's Portfolio</title>
       </head>
 <body>
-    <header> <h1>Jud&aacute; Teixeira</h1> </header>
+  
+  <header>
+    <div>
+      <a href="#" class="logo">
+        <img src="images/banner.png" alt="Jt-logo" height="35">
+      </a>
+    </div>
+    
+
+    <div>
+      <nav>
+        <a href="#" class = "menu">
+        <img src="images/menu.png" alt="Menu" height="35">
+        </a>
+     </nav>
+    </div>
+  </header>
+  
+  <div class="bg">
+    <img src="images/bg.png" alt="BG">
```

Comment:
**Kindly fix this:**
- The section should display the proper background image (decorations are always background images, not img tags).

Reply:
Fixed it, thank you!